docs/DEBUGGING.md

```diff
@@ -32,6 +32,6 @@ Add a new interpreter with `+` and select `From Docker, Vagrant, Vm, Remote...`.
 
 Back in Languages & Frameworks > PHP > Debug ensure the port in the Xdebug section matches the value of `OPG_PHP_XDEBUG_REMOTE_PORT` in `api/docker/env/api.env`.
 
-In Languages & Frameworks > PHP > Test Frameworks add a new configuration type and select `PHP Unit by Remote Interpreter` and select the CLI Interpreter that was just created for Docker api. PHP Unit Library > select `Use Composer Autoloader` and enter `/var/www/vendor/autoload.php` in `Path to script`. Click the refresh symbol and confirm a version of PHP Unit is displayed here - if not then recheck the path mappings set up above. For `Default Configuration File` enter `/var/www/tests/phpunit.xml` and  `Default Bootstrap File` enter `/var/www/tests/bootstrap.php`.
+In Languages & Frameworks > PHP > Test Frameworks add a new configuration type and select `PHP Unit by Remote Interpreter` and select the CLI Interpreter that was just created for Docker api. PHP Unit Library > select `Use Composer Autoloader` and enter `/var/www/vendor/autoload.php` in `Path to script`. Click the refresh symbol and confirm a version of PHP Unit is displayed here - if not then recheck the path mappings set up above. For `Default Configuration File` enter `/var/www/tests/phpunit/phpunit.xml` and  `Default Bootstrap File` enter `/var/www/tests/phpunit/bootstrap.php`.
```

Comment:
Corrected folder structure for the location of **Default Configuration File** and **Default Configuration File**